modules/navigation-tabs/tabbar-icon.js

```diff
@@ -17,6 +17,14 @@ type Props = {
 	focused: boolean,
 }
 
-export const TabBarIcon = (icon: string) => ({tintColor}: Props) => (
-	<Icon name={`ios-${icon}`} style={[styles.icon, {color: tintColor}]} />
-)
+export const TabBarIcon = (icon: string) => ({tintColor}: Props) => {
+	if (icon === 'ionic') {
+		return (
+			<Icon name={`logo-${icon}`} style={[styles.icon, {color: tintColor}]} />
```

Comment:
The "ionic" icon is used in the dev tool for changing the BonApp menu. It should be changed there, not in TabBarIcon.

Reply:
I see your other reply now. I shall think.

Reply:
I propose that we change from "ionic" to "medal", since that's just a random icon that I pulled out for dev builds a long time ago. 

![ebdc9414-cf10-4af1-91b0-1e18aa19422f](https://user-images.githubusercontent.com/464441/52184133-08c29d00-27d5-11e9-9d20-a3d216694d47.png)

Reply:
No.

See what I said about this:

![image](https://user-images.githubusercontent.com/1566689/52184156-54754680-27d5-11e9-99a9-fc01a3341c2f.png)



The current version of `TabBarIcon` prefixes `icon` with `ios-` no matter what its contents were, so if I revert the changes in this file, I would get `ios-logo-ionic`. The only other way I thought of doing this was changing all usages of `TabBarIcon` to include their prefix (i.e. changing `TabBarIcon('blah')` to `TabBarIcon('ios-blah')`) and changing `TabBarIcon` so that it does not prefix its input. Would you prefer that?

Reply:
I want to continue being able to call TabBarIcon with no prefixes. I also want to keep TabBarIcon simple; I have no attachment to the "ionic" icon. 

Thus, I would like to remove the use of the ionic icon from Menus, allowing us to continue using the simple code.